standards-catalogue/source/Guidance/referencearchitecture/index.html.md

```diff
@@ -0,0 +1,369 @@
+---
```

Comment:
All folder should be lowercase (and kebab-case in case of multiwords) so we avoid any surprises with case sensitive vs case insensitive FS.